pdns/dnsdist-lua-actions.cc

```diff
@@ -648,6 +648,47 @@ class DisableECSAction : public DNSAction
   }
 };
 
+class SetECSAction : public DNSAction
+{
+public:
+  SetECSAction(Netmask v4): d_v4(v4), d_hasV6(false)
+  {
+  }
+
+  SetECSAction(Netmask v4, Netmask v6): d_v4(v4), d_v6(v6), d_hasV6(true)
```

Comment:
Couldn't we `const &` constructor arguments (here and with v4 only)?

Reply:
Done!